src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java

```diff
@@ -63,14 +59,22 @@
   private List<WebElement> navigationElements;
   @FindBy(css = ".pi-header")
   private List<WebElement> groupHeadersWrappers;
-  @FindBy(css = ".pi-image")
-  private WebElement imageWrapper;
-  @FindBy(css = ".pi-title")
-  private WebElement titleWrapper;
-  @FindBy(css = "body")
-  private WebElement bodyElement;
-  @FindBy(css = ".header-title")
-  private WebElement headerTitle;
+  @FindBy(css = "#articleCategories .category a")
+  private List<WebElement> categories;
+  @FindBy(css = ".pi-navigation a[href*='redlink']")
+  private List<WebElement> pInfoRedlLink;
```

Comment:
To make it consistent with the following selector "pInfoInternalLinks", lets call it "pInfoRedLinksList" and also add List to "pInfoInternalLinks"